apps/web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@calcom/web",
-  "version": "1.0.0",
+  "version": "1.4.0",
```

Comment:
in order to keep this accurate, we need to change each version for upcoming tagged releases @baileypumfleet

Reply:
we may want to use automatic semantic versioning for this, relying on commit messages to infer the version bump according to the committed work, major, minor and patch.

Reply:
great idea, we can do that in a follow up PR